cli/src/test/resources/ide-projects/mvn/project/workspaces/main/readme

```diff
@@ -0,0 +1 @@
+this is the main workspace of jmc test case
```

Comment:
```suggestion
this is the main workspace of mvn test case
```
BTW: If you just add empty `.gitkeep` files, you avoid such maintenance from copy&paste.